src/Microsoft.AspNetCore.SignalR.Client/HubConnectionFactory.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.SignalR.Internal.Protocol;
+using Microsoft.AspNetCore.Sockets;
+using Microsoft.AspNetCore.Sockets.Client;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.SignalR.Client
+{
+    public static class HubConnectionFactory
+    {
+        public static HubConnection Create(Uri url, ILoggerFactory loggerFactory = null)
```

Comment:
We should also have a Create overload that only takes the Uri. HttpConnection has a constructor that only uses the Uri as well so the loggerFactory isn't needed

Reply:
This is this overload. loggerFactory isn't needed here - it defaults to null if not provided.